nodebuilder/node/config.go

```diff
@@ -0,0 +1,38 @@
+package node
+
+import (
+	"fmt"
+	"time"
+)
+
+var DefaultLifecycleTimeout = time.Minute * 2
```

Comment:
why do you need to define/export this?

Reply:
good catch, it doesnt have to be anymore